src/System.Memory/ref/System.Memory.cs

```diff
@@ -165,6 +165,13 @@ public static class MemoryExtensions
         public static int BinarySearch<T>(this Span<T> span, IComparable<T> comparable) { throw null; }
         public static int BinarySearch<T, TComparable>(this Span<T> span, TComparable comparable) where TComparable : IComparable<T> { throw null; }
         public static int BinarySearch<T, TComparer>(this Span<T> span, T value, TComparer comparer) where TComparer : IComparer<T> { throw null; }
+
+        public static void Sort<T>(this Span<T> span) { throw null; }
```

Comment:
nice: new line.

Generate the reference using our tool.
Run `msbuild /t:GenerateReferenceSource` from the S.Memory source directory.

cc @weshaggard if you have any questions.